visualize-admin/visualization-tool
# feat: Add drag and drop to Dashboard layouts

This PR adds a drag and drop functionality to Dashboard layouts (Vertical and Tall) in the edit mode.

### How to test
1. Go to [this link](https://visualization-tool-git-feat-layout-drag-and-drop-kit-ixt1.vercel.app/en/create/new?cube=https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/9&dataSource=Prod&flag__layoutStep=true).
2. Add two additional charts, so that there are three in total.
3. Proceed to layout options.
4. Change layout to Dashboard.
5. See that there are drag handles next to charts metadata buttons.
6. See that it's possible to drag and drop the charts to change their order.
7. Change layout from Vertical to Tall.
8. See that drag and drop also works for this layout.
